ui/src/text_annotator.tsx

```diff
@@ -113,7 +113,15 @@ const
       top: -3,
       background: cssVar('$card'),
     },
-    readonly: { pointerEvents: 'none' }
+    readonly: { pointerEvents: 'none' },
+    checkbox: {
+      $nest: {
+        '&:hover': {
+          backgroundColor: cssVar('$neutralLighter'),
+          color: cssVar('$neutralDark')
+        }
```

Comment:
I assume this is for consistency with `Remove all`, right?

Reply:
Yes, you're right!